src/app/dashboard/page.tsx

```diff
@@ -152,7 +152,7 @@ const Home: NextPage = () => {
     if (!isFetchedData.current) {
       isFetchedData.current = true
       const res = fetchPrices()
-      const res2 = fetchPriceChange()
+      const res2 = fetchTokenData('IQ')
```

Comment:
we can refactor the names to suit the data better for example here we can call it as `IQTokenData` instead of `res2` and `priceData` or `prices` instead of `res`